pkg/cmd/run.go

```diff
@@ -127,6 +132,18 @@ doppler run --command "YOUR_COMMAND && YOUR_OTHER_COMMAND"`,
 				}
 			}
 
+			if addKey && preserveEnv {
+				// skip key if environment variables already contain this key
+				envKey := fmt.Sprintf("%s=", name)
+				for _, envVariable := range env {
+					if strings.HasPrefix(envVariable, envKey) {
```

Comment:
Why not check if the strings are equal?

Reply:
We don't care if the variables have the same value, just that they share the same name.

Reply:
Oh i see